internal/providers/terraform/aws/testdata/mq_broker_test/mq_broker_test.tf

```diff
@@ -0,0 +1,111 @@
+provider "aws" {
+  region                      = "us-east-1"
+  skip_credentials_validation = true
+  skip_metadata_api_check     = true
+  skip_requesting_account_id  = true
+  skip_get_ec2_platforms      = true
+  skip_region_validation      = true
+  access_key                  = "mock_access_key"
+  secret_key                  = "mock_secret_key"
+}
+
+resource "aws_security_group" "my_aws_security_group" {
+}
+
+resource "aws_mq_configuration" "my_aws_mq_configuration" {
```

Comment:
Might be good to also include ebs and efs storage_type options explicitly to test the prices are ok.